src/lib/Scenes/Fair/Components/HoursCollapsible.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Flex, Sans } from "@artsy/palette"
+import { WhiteButton } from "lib/Components/Buttons"
+import React from "react"
+
+interface Props {
+  hours: string
+}
+
+interface State {
+  isExpanded: boolean
+}
+
+export class HoursCollapsible extends React.Component<Props, State> {
+  state = { isExpanded: false }
+
+  handleToggleIsExpanded = () => {
+    this.setState({ isExpanded: !this.state.isExpanded })
+  }
+
+  render() {
+    const { isExpanded } = this.state
+    const { hours } = this.props
+    return (
+      <Box>
+        <Flex justifyContent="space-between" alignItems="center" flexDirection="row">
+          <Sans size="4">Hours</Sans>
+          <Box width={112}>
```

Comment:
why are we using a hardcoded width here?

Reply:
The way this `Button` works as far as I can tell is that it fills it parent, it's a bit awkward to create a button with horizontal padding. I'll rework this to be wrapped in a Flex constrained by margin so we can remove the hardcoded width.